src/test/groovy/graphql/execution/ExecutionParametersTest.java

```diff
@@ -0,0 +1,41 @@
+package graphql.execution;
+
+import graphql.language.Field;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static graphql.Scalars.*;
+import static graphql.execution.ExecutionParameters.newParameters;
+import static graphql.execution.TypeInfo.newTypeInfo;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonMap;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertSame;
+
+public class ExecutionParametersTest {
```

Comment:
Well.... I know that it's possible ( I'm an official Groovy Committer after all :D ), yet I found it very ugly. But if you still want me to do that then no problem, I can change :)